### PR TITLE
Fix index, remove TeV out from dataset, add FS samples and dictionary

### PR DESCRIPTION
**Pull request by srimanob**

### CODE REVIEW

**Comment by srimanob** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -345,7 +345,9 @@ def identitySim(wf):
     'CMSSW_7_5_0_pre6-PU25ns_75X_mcRun2_asymptotic_v1-v1',  # 5 - fullSim PU 25ns premix
     'CMSSW_7_5_0_pre6-PU50ns_75X_mcRun2_startup_v1-v1',     # 6 - fullSim PU 50ns premix
     'CMSSW_7_5_0_pre6-75X_mcRun2_asymptotic_v1_FastSim-v1', # 7 - fastSim premix
-    'CMSSW_7_5_0_pre6-75X_mcRun2_HeavyIon_v1-v1'            # 8 Run2 HI GEN-SIM
+    'CMSSW_7_5_0_pre6-75X_mcRun2_HeavyIon_v1-v1',           # 8 Run2 HI GEN-SIM
+    'CMSSW_7_5_0_pre5-MCRUN2_75_V5_FastSim-v1',             # 9 - fastSim mb
```
Comment (hengne) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: @srimanob Hi Phat, this parameter is actually the same as parameter " 7 - fastSim premix" , it is repeated. Could you please either remove this one or remove the " 7 - fastSim premix" and replace the things with this "9 - fastSim mb" ? 
Thanks, Hengne

**Comment by hengne** [comment]
@srimanob @boudoul @franzoni @davidlange6  dear all, could we have this PR fixed and merged asap? I have a couple of updates to be put in 76X based on this PR. My updates include updates of the RelVal scripts and adding three new BPH workflows. 
I wish to have all these updates inside the 76x for the next 760_pre3 build. Because these updates includes many changes , if they are not merged in the release fast enough, I need to change everything by hand every time, which can easily waste a couple of hours to make sure everything works properly. 

Thanks in advance, Hengne

**Comment by srimanob** [comment]
close

**Pull request closed without merging**